src/core/less/variables/spacings.less

```diff
@@ -0,0 +1,259 @@
+// Spacings setup
+// ==============
+
+// Basic padding setup
+@item-padding: 20px;
+@component-padding: 20px;
+@block-padding: 60px;
+@article-padding: 60px;
+@page-header-padding: 100px;
+@menu-header-padding: 100px;
+@menu-body-padding: 20px;
+@button-padding: 14px;
+@notify-padding: 60px;
+@navigation-padding: 20px;
+@navigation-padding-mobile: @navigation-padding;
+
+// Basic margin setup
+@item-margin: 5px;
+@title-text-margin: 20px;
+@body-text-margin: 30px;
+@instruction-text-margin: 14px;
+@button-margin: 30px;
+
+
+// Paddings - Detailed Spacings setup
+// ==================================
+
+// Item padding
+@item-padding-top: @item-padding;
```

Comment:
these type of secondary relationships should be broken. a theme will want to/should do these differently. set all to 0.

there should be no or few hierarchical abstractions in the core theme. such that x variable has y impact. sweet and simple. easy to create relationships later.

Reply:
obviously, this might not be possible.